### ISSUE TITLE
BUG: Error processing files with one annotation

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#466]
As reported by @vivinastase here:
https://github.com/yardencsGitHub/tweetynet/issues/204

#### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the past issues](https://github.com/NickleDave/vak/issues)

**Describe the bug**

When processing files with only one annotation, the system raises an error in module crowsetta/validation.py", line 65, in column_or_row_or_1d. This seems to happen because during loading the annotations file in function notmat2annot in module notmat.py, it uses evfuncs.load_notmat, which itself uses loadmat with option squeeze_me=True which makes a one element list into a scalar.

**To Reproduce**
Please include specific commands entered and complete error messages. More specific detail is better.  
Please format code so it easy to read e.g. using triple backticks in [GitHub-flavored Markdown](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)

```
Traceback (most recent call last):
File "/home/vivi/anaconda3/envs/tweetynet/bin/vak", line 8, in
sys.exit(main())
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/vak/main.py", line 45, in main
cli.cli([command](https://github.com/yardencsGitHub/tweetynet/issues/toml_path=config_file)=args.command, config_file=args.configfile)
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/vak/cli/cli.py", line 30, in cli
COMMAND_FUNCTION_MAPcommand
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/vak/cli/prep.py", line 132, in prep
vak_df, csv_path = core.prep(
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/vak/core/prep.py", line 205, in prep
vak_df = dataframe.from_files(
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/vak/io/dataframe.py", line 112, in from_files
annot_list = scribe.from_file(annot_files)
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/crowsetta/notmat.py", line 86, in notmat2annot
notmat_seq = Sequence.from_keyword(labels=np.asarray(list(notmat_dict['labels'])),
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/crowsetta/sequence.py", line 382, in from_keyword
labels) = cls._validate_onsets_offsets_labels(onsets_s,
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/crowsetta/sequence.py", line 267, in _validate_onsets_offsets_labels
onsets_s = column_or_row_or_1d(onsets_s)
File "/home/vivi/anaconda3/envs/tweetynet/lib/python3.8/site-packages/crowsetta/validation.py", line 64, in column_or_row_or_1d
raise ValueError("bad input shape {0}".format(shape))
ValueError: bad input shape ()
```

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Operating System: [Linux, Windows, Mac]
 - Version [e.g. 22]. You can determine this in Python by running `vak.__version__`

**Additional context**

> With regards to the dataset, I was using my own. Here is how the problem-causing annotation file looked like when loaded with loadmat:

{'header': b'MATLAB 5.0 MAT-file Platform: posix, Created on: Thu Mar 10 10:24:41 2022', 'version': '1.0', 'globals': [], 'Fs': array([[44100]]), 'fname': array(['R3406_40911.56229478_1_3_15_37_9.wav'], dtype='<U36'), 'onsets': array([[2336.50793651]]), 'offsets': array([[2449.70521542]]), 'num_sylls': array([[1]]), 'labels': array(['a'], dtype='<U1')}

and here is the same file loaded with evfuncs.load_notmat

{'header': b'MATLAB 5.0 MAT-file Platform: posix, Created on: Thu Mar 10 10:24:41 2022', 'version': '1.0', 'globals': [], 'Fs': 44100, 'fname': 'R3406_40911.56229478_1_3_15_37_9.wav', 'onsets': 2336.5079365079364, 'offsets': 2449.705215419501, 'num_sylls': 1, 'labels': 'a'}

After a patch that makes an array from the scalar, this part of the processing went well.

**Comment by NickleDave** [comment]
Closing as a duplicate of #467

**Comment by NickleDave** [comment]
Re-opening because it's not actually a duplicate!

**Comment by vivinastase** [comment]
Hi Dave

I can confirm that with the new version of evfuncs the loading of notmat files works without error.

Vivi

**Comment by NickleDave** [comment]
Thank you @vivinastase !

I will test as described above and then call this closed.

And I will keep working on those other improvements to make it easier for people to use simple annotation formats.

**Comment by NickleDave** [comment]
Should be fixed by #488

**Comment by NickleDave** [comment]
@all-contributors please add @vivinastase for UserTesting
